Formula/flash.rb

```diff
@@ -0,0 +1,21 @@
+class Flash < Formula
+  desc "Command-line script to flash SD card images of any kind"
+  homepage "https://github.com/hypriot/flash"
+  url "https://github.com/hypriot/flash/releases/download/2.6.0/flash"
+  sha256 "38a8a5793ae5fac138eeff781c932de98977215b7bfd20e5f3e0a7f46920c6d7"
+
+  bottle :unneeded
+
+  def install
+    bin.install "flash"
+  end
+
+  test do
+    system "hdiutil", "create", "-size", "128k", "test.dmg"
+    command = "#{bin}/flash --force --device /dev/disk42 test.dmg"
+    stdout, stderr, status = Open3.capture3(command)
+    assert_equal false, status.success?
+    assert_equal "", stdout
+    assert_match "Could not find disk: /dev/disk42", stderr
```

Comment:
This is the homebrew way.
```suggestion
    output = shell_output("#{bin}/flash --force --device /dev/disk42 test.dmg 2>&1")
    assert_match "Could not find disk: /dev/disk42", output
```

Reply:
Hi Sean,

> This is the homebrew way.

But somehow it doesn't seem to work reliably… 😞

Incidentally, I've actually tried with `shell_output` and various combination of '1', '2', '&' & '>' before resorting to `capture3`, but thought I had mixed up the order again and gave up.  Now it's working locally for me, but apparently not on all versions of OSX.  How do we proceed?